### PR TITLE
Feature/idva5 947 create acsp profile data api skeleton

### PR DESCRIPTION
**Pull request by adeharrison-ch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# acsp-profile-data-api
-acsp-data-api handling the data collection for Authorised Corporate Service Providers
+# `acsp-profile-data-api`
+
+## Summary
+
+The `acsp-profile-data-api` is a service that receives acsp profile deltas from
+`acsp-profile-data-consumer`. It transforms these deltas to a standardised structure and then:
+
+* stores or deletes documents within the `company_acsp_profile collection` in MongoDB, and
```
Comment (cgregoryCH) [inline README.md:8]: Should this be ```acsp_profile```?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+management.endpoints.web.path-mapping.health=healthcheck
+management.endpoint.health.show-details=never
+management.endpoint.health.enabled=true
+management.health.mongo.enabled=false
+
+api.api-url=${CHS_KAFKA_API_URL:http://localhost:8889}
+api.api-key=${ACSP_PROFILE_API_SUB_DELTA_KEY:chsApiKey}
+
+spring.data.mongodb.uri=${MONGODB_URL:mongodb://mongo:27017/acsp_profile_delta}
+spring.data.mongodb.name=acsp_profile_delta
```
Comment (cgregoryCH) [inline src/main/resources/application.properties:12]: I think the database and collection should be "acsp_profile" instead, to be consistent with the company_profile database/collection name, "delta" is generally used for the Kafka topics.

**Review by cgregoryCH — COMMENTED** [review]
Looks good, just a couple of comments about the collection name

File: README.md (hunk 1 above)
Comment (adeharrison-ch) [reply README.md]: done

File: src/main/resources/application.properties (hunk 2 above)
Comment (adeharrison-ch) [reply src/main/resources/application.properties]: done

**Review by cgregoryCH — APPROVED** [review]

**Pull request merged**